Makefile

```diff
@@ -42,6 +45,11 @@ docs: ## Generate documentation.
 	$(MAKE) -C docs clean
 	$(MAKE) -C docs html
 
+.PHONY: rundocs
+rundocs: ## start a documentation runserver
+	. $(DOCSVENV); sphinx-autobuild $(ALLSPHINXOPTS) --ignore ".git/*" --ignore "*.scss" ./docs -b dirhtml -a docs/_build/html --host 0.0.0.0 --port $(PORT)
```

Comment:
Is activating the docs venv necessary here? The instructions have a step to activate it "manually", right?
`make docs` doesn't activate the env (that I can see), and the other makefile commands don't either, so maybe we should skip it

Reply:
I have updated the instructions to reflect this, but yes, I believe that providing the simplest possible path to having getting the docs up and running locally is important to making working on documentation pleasant. I'd have done something similar for `make docs`, but I didn't want to mess with Makefile entries I didn't understand.

Reply:
Alright, then I think we should update `make docs` too for consistency. I've tested locally that the following should work:

```
diff --git a/Makefile b/Makefile
index df0f013..a542d6f 100644
--- a/Makefile
+++ b/Makefile
@@ -42,8 +42,8 @@ coverage: ## Run pytest with coverage report.
 docs: ## Generate documentation.
        rm -f docs/rockcraft.rst
        rm -f docs/modules.rst
-       $(MAKE) -C docs clean
-       $(MAKE) -C docs html
+       . $(DOCSVENV); $(MAKE) -C docs clean
+       . $(DOCSVENV); $(MAKE) -C docs html
```
(If you want I can add those changes on your branch too, I just didn't want to do it without discussing first)

Reply:
Go for it @tigarmo thanks.